### ISSUE TITLE
Create the navigation

### ISSUE THREAD

**Issue by ifr4h** [chingu-voyages/v37-toucans-team-01#19]
**__Task Description__**
This will be repeated throughout the website as the main navigation between the pages. The navigation should include the following:
- [ ] pop up checkout card when the icon is clicked
- [ ] dropdown menu where appropriate
- [ ] hover animations

**__Technical Considerations__**
This should be responsive and accessibility should be considered

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.
